pygfx/controls/_panzoom.py

```diff
@@ -17,9 +17,9 @@ def __init__(
     ) -> None:
         self.rotation = Quaternion()
         if eye is None:
-            eye = Vector3(50.0, 50.0, 50.0)
+            eye = Vector3(0, 0, 0)
         if target is None:
-            target = Vector3()
+            target = Vector3(eye.x, eye.y, eye.z - 100)
```

Comment:
So by default you're looking towards negative z?

Reply:
Exactly!